keyboards/handwired/franky36/config.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2024-2025 Grigory Avdyushin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define HAL_USE_I2C TRUE
+#define I2C_DRIVER I2CD0
+#define I2C1_SDA_PIN GP0
+#define I2C1_SCL_PIN GP1
+
+#define OLED_DISPLAY_128X32
+#define OLED_DISPLAY_WIDTH 128
+#define OLED_DISPLAY_HEIGHT 32
+
+#include_next <mcuconf.h> // for i2c
+#undef RP_I2C_USE_I2C0
+#define RP_I2C_USE_I2C0 TRUE
+#undef RP_I2C_USE_I2C1
+#define RP_I2C_USE_I2C1 FALSE
```

Comment:
This belongs in a `mcuconf.h` file. Please refer to the I2C driver documentation.
```suggestion
```